popup/popup.js

```diff
@@ -39,16 +39,23 @@ chrome.storage.local.get(["websiteUrl"]).then(({ websiteUrl }) => {
 			document.querySelector("#session-image").src = user.image;
 			document.querySelector("#session-name").innerHTML = user.name;
 			document.querySelector("#session-email").innerHTML = user.email;
-
-			chrome.storage.local.set({
-				userId: user.id,
-				userName: user.name,
-				userImage: user.image
-			}).then(() => {
-				console.debug(`[popup] userId has been set to ${user.id}`);
-			}).catch(err => {
-				console.error(`[popup] Sync storage could not be set. userId: ${user.id}`, err);
+			chrome.cookies.get({ url: websiteUrl, name: '__Secure-next-auth.session-token' })
+			.then((cookie) => {
+				const tokenval = cookie.value;
+				chrome.storage.local.set({
+					userId: user.id,
+					userName: user.name,
+					userImage: user.image,
+					token: tokenval,
```

Comment:
Hmm... I was thinking about this earlier when I was dropping the comment. But then I thought that the current CI-based implementation does not actually require the cookie. So in case the fetch-cookie operation fails, we should still be able to proceed.

Are you sure that you don't want the application to show signed-in if the get-cookie operation fails?